src/pages/index.vue

```diff
@@ -23,16 +23,17 @@
       <VSearchBar
         v-model.trim="searchTerm"
         class="max-w-[40rem] mt-4 lg:mt-8"
-        :placeholder="featuredSearchText"
```

Comment:
In the absence of animation, I suppose placing the featured search term as the placeholder is justified?

Reply:
Placeholders get read by the screen reader. Without the visual context of the opposite side of the screen, it would be a completely context-less placeholder and the screen reader would read "Olympic games" and the person relying on a screen reader would have no idea why.